TodoApi/TodoDbContext.cs

```diff
@@ -1,8 +1,15 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿using System.Reflection;
+using Microsoft.EntityFrameworkCore;
 
 public class TodoDbContext : DbContext
 {
     public TodoDbContext(DbContextOptions options) : base(options) { }
 
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        base.OnModelCreating(modelBuilder);
+        modelBuilder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());
```

Comment:
This does assembly scanning, that isn't ideal.

Reply:
I changed ApplyConfigurationsFromAssembly to ApplyConfiguration 
for any entity add configuration to OnModelCreating

Reply:
> Why?

Fluent API provides more functionality for configuration than DataAnnotations
and domain classes clear than any Constant , validation and more